refinery/user_files_manager/tests.py

```diff
@@ -109,6 +109,7 @@ def test_user_files_csv(self):
 
 class UserFilesUtilsTests(TestCase):
 
+    @override_settings(REFINERY_SOLR_DOC_LIMIT=10)
```

Comment:
Can we set the default to 10000 for tests? In general, ten is the solr default and would be unnecessary to include.